tasks/mc/data.py

```diff
@@ -510,6 +510,16 @@ def _update_category(self, session, category):
         session.execute(query)
         session.commit()
 
+    def _remove_underscores(self, session, schema, table, column):
+        if self.country != 'uk' or self.geography not in ['postcode_sector', 'postcode_unit']:
+            return
+
+        session.execute('''
+            UPDATE "{schema}".{table}
```

Comment:
This question may not have much sense, but could this query be done as part of any of the other queries of the ETL?

I'm not sure how expensive is this query to be performed (maybe not too much) but just in case we could save some time 👼

Reply:
I also thought about this, but I think that this is for the best. It's not expensive (seconds, or less). While doing it afterward might look cheap, it's not a problem for a performance point of view, makes it absolutely explicit, and less error-prone (if you check the code, there are several queries where ids are inserted).

@antoniocarlon , do you agree?

Reply:
Given that it's executed only in certain conditions (Postcode sectors and units for the UK) I would not add it to the `insert` to keep the code cleaner there. Also if we have other exceptions in the future we can add them here, so I would keep it